README.md

```diff
@@ -73,6 +73,7 @@ go test -tags pacific ....
 
 | go-ceph version | Supported Ceph Versions | Deprecated Ceph Versions |
 | --------------- | ------------------------| -------------------------|
+| v0.32.0         | pacific, quincy, reef, squid   | octopus           |
```

Comment:
Any changes for squid or octopus?

Reply:
Nothing for this release of go-ceph IMO, but ceph has officially EOLed quincy. Thus we're supporting octopus, pacific, and quincy "legacy releases".  We should decide if/when we want to drop octopus and/or deprecate pacific, but we don't have to make those decisions for this week's release.

Reply:
I meant pacific of course... Alphabet is complicated. 😅